ctk-server/src/main/resources/ctk

```diff
@@ -0,0 +1,95 @@
+#!/bin/bash -e
```

Comment:
Why does this file appear twice? (Once here, once in ctk-cli/src/main/resources/ctk)